shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -223,21 +223,26 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
   NSString* body = notification[@"m"];
   if (type != nil && body != nil) {
     dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^(void){
-      NSLog(@"Remote notification handle for %@ started...", type);
       NSError* err = nil;
       int membersType = [notification[@"t"] intValue];
       if ([type isEqualToString:@"chat.newmessageSilent_2"]) {
         NSString* convID = notification[@"c"];
-        int messageID = [notification[@"d"] intValue];
-        NSString* pushID = [notification[@"p"] objectAtIndex:0];
-        int badgeCount = [notification[@"b"] intValue];
-        int unixTime = [notification[@"x"] intValue];
-        NSString* soundName = notification[@"s"];
-        PushNotifier* pusher = [[PushNotifier alloc] init];
-        KeybaseHandleBackgroundNotification(convID, membersType, messageID, pushID, badgeCount, unixTime, body, soundName, pusher, &err);
+        NSString* msg = nil;
+        msg = KeybaseUnboxNotification(convID, membersType, body, &err);
+        // If d is set, display the plaintext notification
+        bool displayPlaintext = notification[@"n"];
+        if (displayPlaintext) {
```

Comment:
Can we just do this display check in Go just to limit the amount of code in native/JS? I realize it will just be passing in a bool to check for false, but I prefer to have all the logic for this stuff in Go if possible

Reply:
done